src/bento/core/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@bentoproject/core",
+  "author": "Bento Authors",
+  "license": "Apache-2.0",
+  "main": "./dist/bento.core.js",
+  "module": "./dist/bento.core.mjs",
```

Comment:
jw, is this naming convention planned (or already in-use) for all Bento binaries?

Reply:
The "bento.core.*" is a new naming convention I made up for this PR for the Bento Core script. The Bento Runtime is currently already named `bento.js`, so I tried to keep consist with that and named the Bento Core package as `bento.core.js`.

Reply:
I could imagine one being `bento.browser.js` and the other `bento.npm.js`